react-search-component/my-app/src/App.tsx

```diff
@@ -0,0 +1,20 @@
+import { useState } from 'react';
+import SearchableList from './SearchableList';
+import './App.css';
+
+function App() {
+  const [search, setSearch] = useState('');
+
+  return (
+    <div>
+      <input
```

Comment:
If this were inside SearchableList, it would make the SearchableList more reusable -- you could just place the entire component anywhere you need it without having to worry about how to get the input.